doc/apps/shiny-core.md

```diff
@@ -0,0 +1,43 @@
+# Shiny Core
```

Comment:
rename al references from "shiny core" to "shiny"

Reply:
[fixed](https://github.com/ploomber/doc/blob/6318fe9c144499deef0d0550c0626f73f6942aaf/doc/apps/shiny.md?plain=1#L1)
Changed all references